spec/factories/generic_works.rb

```diff
@@ -39,7 +39,7 @@
       additional_title ["Or, There and Back Again q"]
       admin_note    ["This is an admin note"]
       credit_line   ["Courtesy of CHF Collections"]
-      division      "Othmer Library of Chemical History"
+      division      "Othmer"
```

Comment:
gah, I already fixed this but messed up in the rebase somehow! ok thanks for the sharp eye. 

Git gets a lot harder with more than one developer involved!